### PR TITLE
[WIP] on page SEO optimization

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
WIP. Feel free to review/comment.
As soon as everything is done, I'll squash

### CODE REVIEW

**Comment by RiccardoMasutti** [comment]
Structured data results: https://search.google.com/structured-data/testing-tool/u/0/#url=https%3A%2F%2Fraw.githubusercontent.com%2FRiccardoMasutti%2FzkSNACKsSite%2Fpatch-2%2Fwwwroot%2Findex.html

File: wwwroot/index.html [hunk 1]
```diff
@@ -507,10 +765,10 @@ <h4 class="card-title">Max Hillebrand</h4>
 		<footer>
 			Legal notice:
 			<a class="go-to ml-1 mr-1"
-				href="https://zksnacks.com/wp-content/uploads/2018/05/zkSNACKs_Privacy-Policy_vfinal.pdf"
+				href="https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Legal/Assets/LegalDocuments.txt"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html]: @bharmat @hgergoil old file was missing. Can we use the txt file from WalletWasabi repo?

File: wwwroot/index.html [hunk 2]
```diff
@@ -507,10 +765,10 @@ <h4 class="card-title">Max Hillebrand</h4>
...
 				target="_blank"><span style="font-weight: 400;">Privacy policy</span></a>
 			<span class="go-to"> | </span>
-			<a class="go-to ml-1" href="https://zksnacks.com/wp-content/uploads/2018/05/zkSNACKs_Legal.pdf"
+			<a class="go-to ml-1" href="mailto:legal@zksnacks.com"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html]: @bharmat @hgergoil old file was missing. What if we add `mailto:legal@zksnacks.com`?

File: wwwroot/index.html (hunk 1 above)
Comment (bharmat) [reply wwwroot/index.html]: No, @hgergoil is going to make a separate PP for the website.

Comment (RiccardoMasutti) [reply wwwroot/index.html]: #10

**Comment by MaxHillebrand** [comment]
could you please add a screenshot with the visual changes?

File: wwwroot/index.html [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
-	<title>zkSNACKs</title>
+	<title>zkSNACKs - Unfairly Private</title>
 	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
-	<meta name="description" content="Nowadays it is all about individuals’ digital privacy, governments and organisations are
-	making steps to protect privacy in the digital world. We believe that privacy is both
-	a fundamental human right and business need that should be preserved at all times.
-	This is why we focus on your digital financial privacy." />
-	<meta name="keywords"
-		content="bitcoin wallet, protect privacy, wallet, bitcoin mining, bitcoin worth, bitcoin privacy, exchange, store, bitcoin, privacy policy, financial privacy, blockchain, transparent, opensource, crypto, cryptocurrency, currency" />
+	<meta name="description" content="zkSNACKs was established to research the possibilities for Bitcoin privacy enhacements. It's main product is Wasabi, a Bitcoin privacy-oriented wallet." />
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	<meta name="description" content="zkSNACKs was established to research the possibilities for Bitcoin privacy enhancements. Its main product is Wasabi Wallet, a Bitcoin privacy-oriented wallet." />
```

File: wwwroot/index.html [hunk 4]
```diff
@@ -1,17 +1,26 @@
...
+	<meta name="keywords" content="bitcoin wallet, protect privacy, wallet, bitcoin mining, bitcoin worth, bitcoin privacy, exchange, store, bitcoin, privacy policy, financial privacy, blockchain, transparent, opensource, crypto, cryptocurrency, currency" />
```
Comment (yahiheb) [inline wwwroot/index.html:7]: Shouldn't we add `CoinJoin` as a keyword.

File: wwwroot/index.html [hunk 5]
```diff
@@ -1,17 +1,26 @@
...
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<link href='http://fonts.googleapis.com/css?family=Roboto:400,100,300,700,500,900' rel='stylesheet' type='text/css'>
+	<!-- Open Graph / Facebook -->
+	<meta property="og:type" content="website">
+	<meta property="og:url" content="https://zksnacks.com/">
+	<meta property="og:title" content="zkSNACKs - Unfairly Private">
+	<meta property="og:description" content="zkSNACKs was established to research the possibilities for Bitcoin privacy enhacements. It's main product is Wasabi, a Bitcoin privacy-oriented wallet.">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	<meta property="og:description" content="zkSNACKs was established to research the possibilities for Bitcoin privacy enhancements. Its main product is Wasabi Wallet, a Bitcoin privacy-oriented wallet.">
```

File: wwwroot/index.html [hunk 6]
```diff
@@ -1,17 +1,26 @@
...
+	<meta property="og:image" content="https://zksnacks.com/assets/zksnacks.png">
+	<meta property="og:site_name" content="zkSNACKs - Unfairly Private">
+	<!-- Twitter -->
+	<meta property="twitter:card" content="summary_large_image">
+	<meta property="twitter:url" content="https://zksnacks.com/">
+	<meta property="twitter:title" content="zkSNACKs - Unfairly Private">
+	<meta property="twitter:description" content="zkSNACKs was established to research the possibilities for Bitcoin privacy enhacements. It's main product is Wasabi, a Bitcoin privacy-oriented wallet.">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	<meta property="twitter:description" content="zkSNACKs was established to research the possibilities for Bitcoin privacy enhancements. Its main product is Wasabi Wallet, a Bitcoin privacy-oriented wallet.">
```

File: wwwroot/index.html [hunk 7]
```diff
@@ -24,128 +33,335 @@
...
+			"url": "https://zksnacks.com",
+			"logo": "https://zksnacks.com/assets/zksnacks.png",
+				"contactPoint": [
+		{
+			"@type": "ContactPoint",
+			"telephone": "",
+			"contactType": "customer service",
+			"email": "info@zksnacks.com"
+		}
+	],
```
Comment (yahiheb) [inline wwwroot/index.html]: Just a small comment here. the document should be properly formatted.

File: wwwroot/index.html [hunk 8]
```diff
@@ -24,128 +33,335 @@
...
 		<div class="white-block" id="product">
 			<div class="product-container m-auto h-100 text-center">
-				<img class="m-auto" alt="Wasabi Wallet" src="assets/wasabi_wallet.png" style="width:235px">
+				<img class="m-auto" src="assets/wasabi_wallet.png" alt="Wasabi Wallet logo" title="Wasabi Wallet logo" style="width:235px">
 				<div class="card-body">
 					<p class="card-text">
-						zkSNACKs’ flagship product is a privacy oriented Bitcoin wallet, called the Wasabi
-						Wallet. <br>
-						<a class="go-to" href="http://wasabiwallet.io">Go to website.</a>
+						zkSNACKs’ flagship product is a privacy oriented Bitcoin wallet, called the Wasabi Wallet. <br>
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
						zkSNACKs' flagship product is a privacy oriented Bitcoin wallet, called Wasabi Wallet. <br>
```

File: wwwroot/index.html [hunk 9]
```diff
@@ -156,211 +372,99 @@ <h2 class="development-title mt-2 mb-lg-5">Which we are proud of</h2>
...
 							</div>
 						</div>
 					</div>
 					<div class="col-12 col-md-6 col-lg-2 mb-3 pr-0 line6">
 						<div class="card development-card h-100 px-2 py-3">
-							<img class="svg-icon mb-0" src="assets/file.svg" alt="JoinMarket integration">
+							<img class="svg-icon mb-0" src="assets/file.svg" alt="Wasabi downloads" title="Wasabi downloads">
 							<div class="card-body development-card-body">
-								Approximately 170,500 Wasabi downloads
+								Approximately 190,000 Wasabi downloads
```
Comment (yahiheb) [inline wwwroot/index.html]: Soon it will be `200,000` so maybe we can just add that.

File: wwwroot/index.html [hunk 10]
```diff
@@ -24,128 +33,335 @@
...
 			<div class="jumbotron jumbotron-fluid">
 				<div class="container">
-					<div class="mb-4"><img src="assets/zksnacks-logo-white.png" style="width: 425px"> </div>
-					<p class="lead">
-						Nowadays it is all about individuals’ <strong>digital privacy</strong>, governments and organisations are
-						making steps to <strong>protect privacy in the digital world</strong>. We believe that privacy is both
-						a fundamental human right and business need that should be preserved at all times.
-						This is why <strong>we focus on your digital financial privacy</strong>.
+					<div class="mb-4"><img src="assets/zksnacks-logo-white.png" alt="zkSNACKs Logo" title="zkSNACKS Logo" style="width: 425px"> </div>
+					<p class="lead">Nowadays it is all about individuals’ <strong>digital privacy</strong>, governments and organisations are making steps to <strong>protect privacy in the digital world</strong>. We believe that privacy is both a fundamental human right and business need that should be preserved at all times. This is why <strong>we focus on your digital financial privacy</strong>.
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
					<p class="lead">Nowadays it is all about individuals' <strong>digital privacy</strong>, governments and organisations are making steps to <strong>protect privacy in the digital world</strong>. We believe that privacy is both a fundamental human right and business need that should be preserved at all times. This is why <strong>we focus on your digital financial privacy</strong>.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: wwwroot/index.html (hunk 4 above)
Comment (RiccardoMasutti) [reply wwwroot/index.html:7]: I still need to optimize meta keywords, even if in 2020 are pretty useless

**Comment by molnard** [comment]
Is this ready to review?

File: wwwroot/index.html (hunk 9 above)
Comment (yahiheb) [reply wwwroot/index.html]: This is now almost 200,000.

**Comment by RiccardoMasutti** [comment]
> Is this ready to review?

Yes

File: wwwroot/index.html [hunk 11]
```diff
@@ -24,128 +33,335 @@
...
+			"contactType": "customer service",
+			"email": "info@zksnacks.com"
+			}
+			],
+	"sameAs": [
+		"https://twitter.com/Zksnacks_LTD",
+		"https://www.linkedin.com/company/zksnacks/",
+		"https://github.com/zkSNACKs",
+		"https://www.crunchbase.com/organization/wasabi-wallet"
+	]
```
Comment (yahiheb) [inline wwwroot/index.html]: Fix the formatting like this one:
```suggestion
	{
		"@context": "https://schema.org",
		"@type": "Corporation",
		"name": "zkSNACKs",
		"url": "https://zksnacks.com",
		"logo": "https://zksnacks.com/assets/zksnacks.png",
		"contactPoint": [
			{
				"@type": "ContactPoint",
				"telephone": "",
				"contactType": "customer service",
				"email": "info@zksnacks.com"
			}
		],
		"sameAs": [
			"https://twitter.com/Zksnacks_LTD",
			"https://www.linkedin.com/company/zksnacks/",
			"https://github.com/zkSNACKs",
			"https://www.crunchbase.com/organization/wasabi-wallet"
		]
```

File: wwwroot/index.html [hunk 12]
```diff
@@ -24,128 +33,335 @@
...
+		"image": "https://zksnacks.com/assets/contact/greg.jpg",
+		"sameAs": [
+			"https://twitter.com/gergely_hajdu",
+			"https://www.linkedin.com/in/gergelyhajdu/"
+		],
+		"jobTitle": "CEO",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
```
Comment (yahiheb) [inline wwwroot/index.html]: Remove unnecessary white spaces like this one:
```suggestion
		}
```

**Review by yahiheb — CHANGES_REQUESTED** [review]
Conflicts must be fixed.

File: wwwroot/index.html [hunk 13]
```diff
@@ -156,211 +372,99 @@ <h2 class="development-title mt-2 mb-lg-5">Which we are proud of</h2>
...
 							<div class="card-body development-card-body">
-								Approximately 170,500 Wasabi downloads
+								Approximately 200,000 Wasabi downloads
 							</div>
 						</div>
 					</div>
 				</div>
 			</div>
 		</div>
-
```
Comment (molnard) [inline wwwroot/index.html:208]: Our media appearances were considered important by @bharmat and @hgergoil. Are you OK to remove it? From my side, it makes no sense for this to be there like it is now, nobody will click on it.

File: wwwroot/index.html [hunk 14]
```diff
@@ -156,211 +372,99 @@ <h2 class="development-title mt-2 mb-lg-5">Which we are proud of</h2>
...
 							</div>
 						</div>
 					</div>
 					<div class="col-12 col-md-6 col-lg-2 mb-3 pr-0 line6">
 						<div class="card development-card h-100 px-2 py-3">
-							<img class="svg-icon mb-0" src="assets/file.svg" alt="JoinMarket integration">
+							<img class="svg-icon mb-0" src="assets/file.svg" alt="Wasabi downloads" title="Wasabi downloads">
 							<div class="card-body development-card-body">
-								Approximately 170,500 Wasabi downloads
+								Approximately 200,000 Wasabi downloads
```
Comment (molnard) [inline wwwroot/index.html]: Maybe a link to https://somsubhra.com/github-release-stats/?username=zksnacks&repository=WalletWasabi

**Review by molnard — COMMENTED** [review]
Tested ACK - consider my comments. It is a huge work well done!

File: wwwroot/index.html (hunk 13 above)
Comment (bharmat) [reply wwwroot/index.html:208]: Agreed, we already discussed it with Riccardo to get rid of it.

File: wwwroot/index.html (hunk 14 above)
Comment (bharmat) [reply wwwroot/index.html]: Isnt some kind of a counter possible? :)

File: wwwroot/index.html (hunk 13 above)
Comment (molnard) [reply wwwroot/index.html:208]: ACK

File: wwwroot/index.html (hunk 14 above)
Comment (molnard) [reply wwwroot/index.html]: Not really. Different releases, also the signatures should be excluded - so it is complex.

**Comment by RiccardoMasutti** [comment]
@yahiheb @molnard should be ready

File: wwwroot/index.html [hunk 15]
```diff
@@ -24,124 +33,332 @@
...
+		"sameAs": [
+			"https://twitter.com/gergely_hajdu",
+			"https://www.linkedin.com/in/gergelyhajdu/"
+		],
+		"jobTitle": "CEO",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+		}
```
Comment (yahiheb) [inline wwwroot/index.html]: The code is still not well formatted.

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by RiccardoMasutti** [comment]
@yahiheb json should be well formatted now

File: wwwroot/index.html [hunk 16]
```diff
@@ -24,124 +33,329 @@
...
+		}],
+		"sameAs": [
+			"https://twitter.com/Zksnacks_LTD",
+			"https://www.linkedin.com/company/zksnacks/",
+			"https://github.com/zkSNACKs",
+			"https://www.crunchbase.com/organization/wasabi-wallet"
+		]
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 17]
```diff
@@ -24,124 +33,329 @@
...
+			"https://www.linkedin.com/in/gergelyhajdu/"
+		],
+		"jobTitle": "CEO",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 18]
```diff
@@ -24,124 +33,329 @@
...
+			"https://www.linkedin.com/in/balintharmat//"
+		],
+		"jobTitle": "CEO",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 19]
```diff
@@ -24,124 +33,329 @@
...
+			"https://github.com/nopara73"
+		],
+		"jobTitle": "Head of Research",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 20]
```diff
@@ -24,124 +33,329 @@
...
+			"https://github.com/lontivero"
+		],
+		"jobTitle": "Lead Engineer",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 21]
```diff
@@ -24,124 +33,329 @@
...
+			"https://github.com/molnard"
+		],
+		"jobTitle": "CTO",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 22]
```diff
@@ -24,124 +33,329 @@
...
+			"https://github.com/danwalmsley"
+		],
+		"jobTitle": "Developer",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 23]
```diff
@@ -24,124 +33,329 @@
...
+			"https://github.com/yahiheb"
+		],
+		"jobTitle": "Developer",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 24]
```diff
@@ -24,124 +33,329 @@
...
+			"https://github.com/jmacato/"
+		],
+		"jobTitle": "Developer",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 25]
```diff
@@ -24,124 +33,329 @@
...
+			"https://github.com/Rmartin1104"
+		],
+		"jobTitle": "Junior Developer",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 26]
```diff
@@ -24,124 +33,329 @@
...
+			"https://www.linkedin.com/in/dan-beluska-a68a63a4/"
+		],
+		"jobTitle": "Head of Marketing",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type="application/ld+json">
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type="application/ld+json">
```

File: wwwroot/index.html [hunk 27]
```diff
@@ -24,124 +33,329 @@
...
+			"https://github.com/MaxHillebrand"
+		],
+		"jobTitle": "Key account manager",
+		"worksFor": {
+			"@type": "Organization",
+			"name": "zkSNACKs"
+		}
+	}
+		</script>
+		<script type='application/ld+json'>
```
Comment (yahiheb) [inline wwwroot/index.html]:
```suggestion
	</script>
	<script type='application/ld+json'>
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by molnard — APPROVED** [review]

**Pull request merged**